crates/ruff_linter/src/rules/flake8_pytest_style/rules/fixture.rs

```diff
@@ -930,9 +925,7 @@ pub(crate) fn fixture(
             check_fixture_decorator(checker, name, decorator);
         }
 
-        if checker.enabled(Rule::PytestDeprecatedYieldFixture)
-            && checker.settings.flake8_pytest_style.fixture_parentheses
```

Comment:
I think that was a bug. It's unclear to me why this rule would depend on enabling that setting.

Reply:
Yeah, that's odd. I tried digging through blame but the best I can conclude without spending too much time on it is that it's been this way for a long time xD